roles/common/defaults/main.yml

```diff
@@ -14,6 +14,9 @@ common:
   ntpd:
     servers: []
     #  - servertime.service.softlayer.com
+  serial_console:
```

Comment:
serial_console is already defined much further down the file. The second definition is replacing this first definition and causing the conditional to fail.